src/LightcyclerSampleSheet/RelativeQuantificationSample.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\LightcyclerSampleSheet;
+
+use MLL\Utils\Microplate\Coordinates;
+use MLL\Utils\Microplate\CoordinateSystem12x8;
+
+class RelativeQuantificationSample
+{
+    public string $sampleName;
+
+    public string $filterCombination;
+
+    public string $hexColor;
+
+    /** @var Coordinates<CoordinateSystem12x8>|null */
+    public ?Coordinates $replicationOf;
+
+    /** @param Coordinates<CoordinateSystem12x8>|null $replicationOf */
+    public function __construct(
+        string $sampleName,
+        string $filterCombination,
+        string $hexColor,
+        ?Coordinates $replicationOf
+    ) {
+        $this->sampleName = $sampleName;
+        $this->filterCombination = $filterCombination;
+        $this->hexColor = $hexColor;
+        $this->replicationOf = $replicationOf;
+    }
+
+    /** @return list<string> */
+    public function serialize(string $coordinatesString): array
```

Comment:
`serialize` steht in PHP üblicherweise für die Konvertierung zu einem `string`. Selbst breiter definiert ist zumindest die Rede davon dass Serialisierung zu einem Stream von Bits oder Bytes führt. Ich finde das passt hier nicht, daher schlage ich einen der folgenden Namen vor:

- `toArray`
- `toSerializableArray`